mscore/workspace.cpp

```diff
@@ -1239,6 +1282,7 @@ Workspace* WorkspacesManager::createNewWorkspace(const QString& name)
       w->setDirty(false);
       w->setReadOnly(false);
       w->write();
+      w->setSourceWorkspaceName(WorkspacesManager::currentWorkspace()->sourceWorkspace()->translatableName());
```

Comment:
`sourceWorkspaceName` could probably be used directly here, although the current code should work as well